irteus/irtutil.l

```diff
@@ -368,6 +368,19 @@
           (reverse ret))
       (close strm))))
 
+(defun make-robot-model-from-name (name &rest args)
+  "make a robot model from string: (make-robot-model \"pr2\")"
+  (let ((name (string-downcase name)))
+    (let ((klasses (flatten (send robot-model :hierarchy))))
+      (dolist (klass klasses)
+        (let ((klass-name (string-downcase (send klass :name))))
+        (if (or (string= klass-name name)
```

Comment:
in irteus, robot like `cmu-bvh-robot-model` is defined. I don't mind support only [robot-name]-model or both of [robot-name]-model and [robot-name]-robot-model.